k8s/upgrades/README.md

```diff
@@ -26,8 +29,9 @@ The upgrade of OpenEBS is a three step process:
 
 **Note: It is mandatory to make sure to that all OpenEBS control plane 
 and data plane components are running with expected version before the upgrade.
-- For upgrading to 1.1.0, the previous version should be 1.0.0
+- For upgrading to 1.1.0, the previous version should be 1.0.0**
```

Comment:
_n: the ** should be only on the last line. this will render entire section "Note" as bold.

Reply:
Added ** for all list items as the formatting was not making all the text bold.